use-persisted-state.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module 'use-persisted-state' {
+  function createPersistedState<T>(key: string, storage: Storage): (initialState: T) => [S, Dispatch<SetStateAction<S>>];
+  export = createPersistedState;
```

Comment:
since this is the default export, I had to use `export default createPersistedState`. Without that, it would work fine, but TS was complaining about synthetic default imports